tests/test_cli.py

```diff
@@ -139,3 +190,50 @@ def test_triton_profile(self, mocker, monkeypatch):
         args = parse_args()
         args.func(args)
         mock_run.assert_called_once_with(["genai-perf", "-m", "add_sub"], check=True)
+
+    @pytest.mark.parametrize("model", ["mock_llm"])
+    def test_triton_metrics(self, model, setup_and_teardown):
+        # Import the Model Repo
+        pid = utils.run_server(repo=MODEL_REPO)
+        setup_and_teardown.pid = pid
+        utils.wait_for_server_ready()
+
+        metrics_before = self._metrics()
+
+        # Before Inference, Verifying Inference Count == 0
+        for loaded_models in metrics_before["nv_inference_request_success"]["metrics"]:
+            if loaded_models["labels"]["model"] == model:  # If mock_llm
+                assert loaded_models["value"] == 0
+
+        # Model Inference
+        self._infer(model, prompt=PROMPT)
+
+        metrics_after = self._metrics()
+
+        # After Inference, Verifying Inference Count == 0
+        for loaded_models in metrics_after["nv_inference_request_success"]["metrics"]:
+            if loaded_models["labels"]["model"] == model:  # If mock_llm
+                assert loaded_models["value"] == 1
+
+    @pytest.mark.parametrize("model", ["mock_llm"])
+    def test_triton_config(self, model, setup_and_teardown):
+        # Import the Model
+        pid = utils.run_server(repo=MODEL_REPO)
+        setup_and_teardown.pid = pid
+        utils.wait_for_server_ready()
+
+        config = self._config(model)
+
+        # Checks if correct model is loaded
+        assert config["name"] == model
+
+    @pytest.mark.parametrize("model", ["mock_llm"])
+    def test_triton_status(self, model, setup_and_teardown):
+        pid = utils.run_server(repo=MODEL_REPO)  # Import the Model
+        setup_and_teardown.pid = pid
+        utils.wait_for_server_ready()
```

Comment:
For this PR, this is good - don't change it here. 

However, some future food for thought - these tests that do (1) server start + (2) utility command should probably be in `test_e2e.py`. For `test_cli.py` it would probably be nice to have some super simple mock server implementation that can be used to test each of these commands standalone (more unit test style).